src/content/drafts/a-beginners-guide-to-eslint.md

```diff
@@ -0,0 +1,320 @@
+
+The guide is aimed at the following audiences:
+
+* JavaScript (ECMASCRIPT) developers new to ESLint
+* Junior developers who want to write more consistent JavaScript
+
+This guide assumes that you are familiar with the following programming tools/languages:
+
+* basic JavaScript.
+* Node.js
+* npm/ yarn
+* using the command line
+
+The code examples in this guide will use `npm`. If you are using `yarn`, make sure to run the corresponding commands.
+
+At the end of this guide you will understand what ESLint is, why it is useful and how to integrate it in your own JavaScript projects and workflows. Additionally this guide will provide paths and links to further resources that will deepen your knowledge of ESLint and result in you writing more robust and bug free code. 
+
+## What is ESLint?
+
+ESlint is an open source linter that analyzes JavaScript code and identifies patterns in the code. This helps you find and fix potential bugs and results in more consistent and maintainable code.
+
+ESLint is highly flexible, it allows you to define your own rules, extend rules made by others and disable rules for specific folders, files, and lines of code.
+
+### What can you do with ESLint?
+
+Linters help you comply with common - or at least agreed upon- best practices. Using a tool like ESLint frees teams (and even individual contributors) from getting caught up in minor details of code style and catches probable bugs early, allowing developers to focus on real problems. The resulting code is more consistent and easier to maintain.
+
+Below are some example ESLint use cases:
+
+* warn if there are unused variables and imports in your code
+* warn when `let` is used for unchanging variables
+* enforce a maximum number of parameters in function definitions
+* warn if a promise is dangling (this causes many bugs)
+* checking for exposed secrets/credentials
+* catch problematic code patterns, for example using loose equality operators (`==` and `!=`) instead of strict equality operators (`===` and `!==`)
+* prevent type conversion
+* warn of unreachable code, for example code that comes after `return` or `break`
+
+A good linter like ESLint will free you from focusing on trivial details, force you to adopt programming best practices, and help you catch bugs while at the same time retaining the flexibility to choose *when* to violate those rules.
+
+## Installation
+
+Navigate to the root of your project, and if you have not already, initialize it with `npm` or `yarn`:
+
+```shell
+npm init
+```
+
+This will generate a `package.json` file. Now run:
+
+```shell
+npm init @eslint/config
+```
+
+This will walk you through configuring your ESLint settings. You will be asked about the the environment, whether you are using React, if you intend to use ESLint for code formatting and other options.
```

Comment:
```suggestion
This will walk you through configuring your ESLint settings. You will be asked about the environment, whether you are using React if you intend to use ESLint for code formatting, and other options.
```